### PR TITLE
VB-2060 - Extend visit and visitor dto to return full details

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
Extend visit and visitor dto to return full details

### CODE REVIEW

**Review by AledWEvans — COMMENTED** [review]
Some note I would just return the details the ui needs and call it a summary of some form VisitSummaryDto

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/FullVisitDetailsDto.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.enums.OutcomeStatus
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.enums.VisitRestriction
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.enums.VisitStatus
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.enums.VisitType
+import java.time.LocalDateTime
+
+@Schema(description = "Full Visit Details")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class FullVisitDetailsDto(
```
[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/FullVisitDetailsDto.kt]: Update to VisitSummaryDto as discussed and only returning the data that is needed.

**Review by AledWEvans — DISMISSED** [review]
LGTM

**Review by AledWEvans — COMMENTED** [review]
LGTM

**Review by AledWEvans — APPROVED** [review]
LGTM, please dont release this until my change

**Pull request merged**